composer.json

```diff
@@ -21,11 +21,20 @@
         "sensio/framework-extra-bundle": "~3.0,>=3.0.2",
         "sensio/generator-bundle": "~2.3",
         "incenteev/composer-parameter-handler": "~2.0",
```

Comment:
incenteev/composer-parameter-handler can build parameters.yml using environment variables afaik, maybe it could be used instead of parameters.php?